docs/ref/forms/models.txt

```diff
@@ -1,15 +1,275 @@
-====================
-Model Form Functions
-====================
-
-Model Form API reference. For introductory material about model forms, see the
-:doc:`/topics/forms/modelforms` topic guide.
+================================================
+ModelForm ``Meta`` Options and Factory Functions
+================================================
 
 .. module:: django.forms.models
-   :synopsis: Django's functions for building model forms and formsets.
+    :synopsis: ModelForm API reference for inner ``Meta`` class and factory
+        functions
+
+.. admonition:: Other ModelForm-related documentation
+
+    This document is an API reference for the :class:`~django.forms.ModelForm`
+    class. For examples or additional context, please refer to the
+    documentation topic :doc:`Creating forms from models
+    </topics/forms/modelforms>`.
```

Comment:
I believe the previous note was sufficient. There's no need for an admonition here.

```
Model Form API reference. For introductory material about model forms, see the
:doc:`/topics/forms/modelforms` topic guide.
```